error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/AssertJInstantRules.java

```diff
@@ -0,0 +1,117 @@
+package tech.picnic.errorprone.refasterrules;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.errorprone.refaster.annotation.AfterTemplate;
+import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import java.time.Instant;
+import org.assertj.core.api.AbstractBooleanAssert;
+import org.assertj.core.api.AbstractInstantAssert;
+import tech.picnic.errorprone.refaster.annotation.OnlineDocumentation;
+
+/**
+ * Refaster rules related to AssertJ assertions over {@link Instant}s.
+ *
+ * <p>These rules simplify and improve the readability of Instant assertions by using the more
+ * specific AssertJ Instant assertion methods instead of generic assertions.
+ */
+@OnlineDocumentation
+final class AssertJInstantRules {
+  private AssertJInstantRules() {}
+
+  static final class AbstractInstantAssertIsAfter {
+    @BeforeTemplate
+    AbstractBooleanAssert<?> before(Instant instant, Instant other) {
```

Comment:
We're not superconsistent yet with our parameter naming, but I expect that eventually we'll start deriving those from the naming "dictated" by the `@AfterTemplate`, where possible. So here I'll rename `instant` to `actual`.